### PR TITLE
ghostscript 9.18

### PR DESCRIPTION
**Pull request by dunn**
Replacing the vendored dependencies with our own was causing problems; see https://github.com/Homebrew/homebrew/issues/43329

### CODE REVIEW

**Comment by dunn** [comment]
Forgot to check revisions but looks like libspectre isn't compatible yet anyway: https://gist.github.com/dunn/e2bb64b63db6de7ec04f#file-02-make-L16-L30

**Comment by dunn** [comment]
https://bugs.freedesktop.org/show_bug.cgi?id=92637

**Comment by dunn** [comment]
Yeah, but it also needs patching.  Should we use the Gentoo one (https://gitweb.gentoo.org/repo/gentoo.git/tree/app-text/libspectre/files/libspectre-0.2.7-gs918.patch?id=8d34151aea08aa04cca35e1f5da59445af0440d0) or wait for upstream?

**Comment by mikemcquaid** [comment]
Happy to use a patch that's been submitted upstream (as usual) :+1:

**Comment by dunn** [comment]
OK, the change to using all the vendored deps is a separate commit now, so it can be reverted if it does cause trouble.

File: Library/Formula/ghostscript.rb [hunk 1]
```diff
@@ -27,25 +22,16 @@ class Ghostscript < Formula
...
-  depends_on "jpeg"
-  depends_on "libtiff"
-  depends_on "jbig2dec"
-  depends_on "little-cms2"
-  depends_on "libpng"
-  depends_on :x11 => ["2.7.2", :optional]
   depends_on "djvulibre" if build.with? "djvu"
-  depends_on "freetype"
+  depends_on "little-cms2"
+  depends_on :x11 => :optional
```
Comment (dunn) [inline Library/Formula/ghostscript.rb]: The 2.7.2 requirement was from almost three years ago (#16924) and I'm guessing it's not necessary?

**Comment by dunn** [comment]
Finally replicated that build failure locally. Maybe I still had GS 9.16 installed and that's where it was finding `gserrors.h`.

**Comment by dunn** [comment]
Nevermind, even weirder: we're only installing that header when `--HEAD`.

**Comment by dunn** [comment]
Aha http://git.ghostscript.com/?p=ghostpdl.git;a=commit;h=feafe5e540a0545ec5d28f3f66bb542056bba495

**Pull request closed without merging**